test/ecto_ordered_test.exs

```diff
@@ -337,16 +328,11 @@ defmodule EctoOrderedTest.Scoped do
 
     model2 |> Model.changeset(%{scoped_position: 4, scope: 2}) |> Repo.update
 
-    assert Repo.get(Model, model1.id).scoped_position == 1
     assert Repo.get(Model, model1.id).scope == 1
-    assert Repo.get(Model, model3.id).scoped_position == 2
     assert Repo.get(Model, model3.id).scope == 1
+    assert ranked_ids(Model, 1) == [model1.id, model3.id]
 
-    assert Repo.get(Model, xmodel1.id).scoped_position == 1
-    assert Repo.get(Model, xmodel2.id).scoped_position == 2
-    assert Repo.get(Model, xmodel3.id).scoped_position == 3
-    assert Repo.get(Model, model2.id).scoped_position == 4
-    assert Repo.get(Model, model2.id).scope == 2
+    assert ranked_ids(Model, 2) == [xmodel1.id, xmodel2.id, xmodel3.id, model2.id]
```

Comment:
Fair point. I'll re-name these to make this more clear. (It's a while since I authored this, but I'm guessing it was a holdover from the existing tests which I haven't got around to refactoring)